collection_mutation.cc

```diff
@@ -153,6 +153,27 @@ api::timestamp_type collection_mutation_view::last_update(const abstract_type& t
     ));
 }
 
+std::ostream& operator<<(std::ostream& os, const collection_mutation_view::printer& cmvp) {
+    fmt::print(os, "{{collection_mutation_view ");
+    cmvp._cmv.with_deserialized(cmvp._type, [&os, &type = cmvp._type] (const collection_mutation_view_description& cmvd) {
+        auto&& ctype = static_cast<const collection_type_impl&>(type);
+        auto&& key_type = ctype.name_comparator();
+        auto&& value_type = ctype.value_comparator();
```

Comment:
I'm afraid this doesn't work for non-frozen UDTs, where value types depend on the keys. Check out `collection_mutation_view_description::materialize` for an example on how to deal with UDTs:
```
        for (auto&& e : cells) {
            m.cells.emplace_back(to_bytes(e.first), atomic_cell(*utype.type(deserialize_field_index(e.first)), e.second));
        }
```

Reply:
I knew that conflating UDTs with collections will hit us. I'll try to fix it.

Reply:
I opened an issue:
https://github.com/scylladb/scylla/issues/5534